rare/ui/components/tabs/settings/legendary.ui

```diff
@@ -48,7 +48,7 @@
            </sizepolicy>
           </property>
           <property name="text">
-           <string>Max sorkers</string>
+           <string>Max workers</string>
```

Comment:
Oh lol, XD.

Can you regen the `py` file too and add it to the PR so it is complete?